### PR TITLE
Make entry.sh more portable by removing all bashisms

### PR DESCRIPTION
**Pull request by bbedward**
The docker `entry.sh` script cannot be run in an environment where bash is not present.

This removes all the bashisms to make the script more compatible with more embeddable environments.

An example of why this is might be useful is if you want to use an alpine base image to run a nano node, you can get slightly faster build times and save ~4Mb by not installing bash

### CODE REVIEW

**Comment by MacroChip** [comment]
Should likes 16 and 18 then be "sh pass through"?

File: docker/node/entry.sh [hunk 1]
```diff
@@ -1,72 +1,72 @@
...
+	printf "  $0 nano_node [daemon] [cli_options] [-l] [-v size]\n"
+	printf "    daemon\n"
+	printf "      start as daemon\n\n"
+	printf "    cli_options\n"
+	printf "      nano_node cli options <see nano_node --help>\n\n"
+	printf "    -l\n"
+	printf "      log to console <use docker logs {container}>\n\n"
+	printf "    -v<size>\n"
+	printf "      vacuum database if over size GB on startup\n\n"
+	printf "  $0 bash [other]\n"
```
Comment (argakiig) [inline docker/node/entry.sh]: remove 16-18 or reference sh

File: docker/node/entry.sh [hunk 2]
```diff
@@ -1,72 +1,72 @@
...
+				echo "log_to_cerr = true"
+				command="${command} --config"
+				command="${command} node.logging.log_to_cerr=true"
+				;;
+			*)
+			 	command="${command} $i"
+				;;
+		esac
 	done
-elif [[ "${TEMP_OPTS[0]}" = 'bash' ]]; then
```
Comment (argakiig) [inline docker/node/entry.sh]: was this intentionally removed without replacing with a sh alternative? see previous comment

**Comment by bbedward** [comment]
Re-added passthrough option

image

**Pull request closed without merging**